### PR TITLE
[Hue] - CDPD-32891 - Cloud Break Template change for QP

### PR DESCRIPTION
**Pull request by Amlesh1902**
Testing: I have done end-to-end testing and found that it's working as expected.

### CODE REVIEW

File: core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java [hunk 1]
```diff
@@ -367,7 +371,8 @@ void getKnoxExposedServicesNamesWith7215() {
             "QUEUEMANAGER_WEBAPP",
             "KNOX_TOKEN_INTEGRATOR",
             "KAFKA_CONNECT",
-            "SSB-SSE-API");
+            "SSB-SSE-API",
+                "QUERY_PROCESSOR");
```
Comment (topolyai5) [inline core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java:375]: intention error

**Review by topolyai5 — CHANGES_REQUESTED** [review]

**Comment by doktoric** [comment]
is this required for 7.2.14 ? You mentioned it only requires 7.2.15

**Comment by Amlesh1902** [comment]
@sreenaths , I think it should be for 7.2.15, please confirm about 7.2.14.
@doktoric we will get back in sometime with the answer about 7.2.14. I hope thereafter we are good to merge this in CB-2.55.0.

File: core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java (hunk 1 above)
Comment (sreenaths) [reply core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java]: Done

**Comment by sreenaths** [comment]
@Amlesh1902 @doktoric 
As you can see in the CSD, the service is supported from CDH v7.2.13 and above. https://github.infra.cloudera.com/Starship/cmf/blob/cdpd-master/csd/QUERY_PROCESSOR/src/descriptor/service.sdl#L13
As 7.2.13 was scrapped, we went with 7.2.14.
But other than that there is no hard requirement for for 7.2.14.

Support from 7.2.15 is good enough.

**Comment by topolyai5** [comment]
Could you please squash the commits and resolve the conflicts?

**Comment by doktoric** [comment]
please rebase

**Comment by lnardai** [comment]
Close the PR since it looks abandoned. Please reopen and rebase it if it's still required.

**Pull request closed without merging**